One to one field custom related name fix

frwickst opened the pull request:
The code is almost the same as @aemdy fix, https://github.com/aemdy/django-reversion-compare/commit/da60d7873bbd8fb212f7f076a68b0b52e297796e. Just a bit more nicely structured and with a proper commit message.

This should fix: #36

Note that there are other problems in the code base that will most likely cause a comparison to still fail, such as #41.

jedie commented:
This is general a not good tested code part.
See: https://coveralls.io/builds/2847195/source?filename=reversion_compare%2Fadmin.py#L172

Can you add unittests for this?!?

jedie commented on reversion_compare/admin.py:
```diff
@@ -106,9 +107,15 @@ def get_related(self):
 
     def get_reverse_foreign_key(self):
         obj = self.version.object_version.object
-        # self = getattr(obj, self.field.related_name) #self.field.field_name
```
If this line is not needed anymore, it should be deleted.

frwickst commented:
While writing the test I noticed that the problem was not actually having a OneToOne field, but a combination of a OneToOne field and having a custom related name on that field.

I modified the branch and added a failing test (that works after the fix).

jedie commented:
> While writing the test I noticed that the problem was not actually having a OneToOne field...

Again a new confirmation for writing unittest :+1: ;)

frwickst commented:
Any plans on merging this?

jedie commented:
Thanks for the reminder!

jedie commented:
v0.5.4 release is out.
